public/config.js

```diff
@@ -1,16 +1,16 @@
 window.ENV = {
   auth: {
-    "domain": "yourtenant.auth0.com",
-    "clientId": "your-cliend-id",
+    "domain": "athenian-development.auth0.com",
+    "clientId": "oodpoYJUvWPyjjsrXjEsSgwE7PROni2r",
     "redirectUri": "http://localhost:3000/login/callback",
     "logoutRedirectUri": "http://localhost:3000",
-    "audience": "https://yourtenant.auth0.com/api/v2/"
+    "audience": "http://localhost:8080"
```

Comment:
according to the previous value, shouldn't this use the new tenant?
```suggestion
    "audience": "https://athenian-development.auth0.com/api/v2/"
```

Reply:
That value is correct already because it expects the api to be running locally at `localhost:8080` so also the name of the audience is `http://localhost:8080`.

Reply:
then, I don't remember why did we need to define the `audience` as `https://<yourtenant>.auth0.com/api/v2/`. I remember that I had some problems when using `localhost:8080`, but since you already tested this, let's merge as it is.